modules/management/kubectl-apply/manifests/kueue-v0.11.4.yaml

```diff
@@ -11426,6 +11426,14 @@ metadata:
     control-plane: controller-manager
   name: kueue-manager-role
 rules:
+- apiGroups:
+    - batch
```

Comment:
Let's open a feature request on [Kueue project](https://github.com/kubernetes-sigs/kueue/tree/main) to enable cronjob support via the manifest.